docs/reference.rst

```diff
@@ -891,3 +891,45 @@ Stores the number of milliseconds to wait between request retries.  By default,
 
 In the case that both this environment variable and the config's ``rpc_attempt_interval`` property are set, the value in ``rpc_attempt_interal`` will be used.
 
+************
+Localization
+************
+
+Shotgun API offers the ability to return display namnes in user's language.
+The server currently returns localized display names for those methods:
```

Comment:
I suggest changing this line (and removing the two bullet points below) to "This functionality is currently supported by the methods `schema_entity_read` and `schema_field_read`."